### ISSUE TITLE
Report does not show bug status for bugs that don't have 

### ISSUE THREAD

**Issue by bladernr** [canonical/lp-to-jira#26]
https://bugs.launchpad.net/plainbox-provider-certification-server/+bug/1662693
SERVCERT-326 | LP#1662693 [None] precheck script crashes if non-USB removab | Backlog                  | 1662693      | 6    |            | none       |              |        |         |       |        |        |       

In the above bug, which was pulled in when I did a project sync, lp-to-jira-report is not showing the bug status.  The bug is marked Fix Released for the package, but I suspect that it's just recognizing those statuses from Release tasks, and doesn't account for things that don't have Release tasks assigned.

**Comment by mclemenceau** [comment]
yes I've seen this problem before I believe this is because lack of project task in Launchpad, Since this is fairly common, probably something that need fixing
